packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.spec.js

```diff
@@ -1,72 +1,216 @@
-          theme={{ dark: false }}
-        />
+  before(function () {
+    routerMock = sinon.stub(Router, 'useRouter').callsFake(() => {
+      return {
+        asPath: 'projects/foo/bar',
+        push: () => {},
+        prefetch: () => new Promise((resolve, reject) => {}),
+        query: { owner: 'foo', project: 'bar' }
+      }
+    })
+  })
+
+  after(function () {
+    routerMock.restore()
+  })
+
+  describe('ProjectAboutPageConnector', function () {
+    it('should render without crashing', function () {
+      const output = render(
+        <Provider store={mockStore}>
+          <Grommet theme={zooTheme} themeMode='light'>
+            <ProjectAboutPageConnector pageType='science_case' />
+          </Grommet>
+        </Provider>
       )
+      expect(output).to.be.ok()
     })
 
-    it('should render without crashing', () => {
-      expect(wrapper).to.be.ok()
+    it('should pass correct data to ProjectAboutPage depending on pageType', function () {
+      const { getByText, getByRole, queryByRole } = render(
+        <Provider store={mockStore}>
+          <Grommet theme={zooTheme} themeMode='light'>
+            <ProjectAboutPageConnector pageType='science_case' />
+          </Grommet>
+        </Provider>
+      )
+      const content = getByText(mockStore.project.about_pages[0].content)
+      expect(content).to.exist()
+
+      expect(getByRole('heading', { name: 'Research' })).to.exist()
+      expect(queryByRole('heading', { name: 'Research Case' })).to.not.exist()
     })
 
-    it('should render the dropdown nav on mobile screen sizes', () => {
-      wrapper.setProps({ screenSize: 'small' })
-      expect(wrapper.find(AboutDropdownNav)).to.have.lengthOf(1)
+    it('should pass default content if a page doesnt exist yet', function () {
+      const { getByText, getByRole } = render(
+        <Provider store={mockStore}>
+          <Grommet theme={zooTheme} themeMode='light'>
+            <ProjectAboutPageConnector pageType='team' />
+          </Grommet>
+        </Provider>
+      )
+      expect(getByRole('heading', { name: 'The Team' })).to.exist()
+      expect(getByText('No content yet.')).to.exist()
     })
 
-    it('should render the sidebar nav on desktop screen sizes', () => {
-      wrapper.setProps({ screenSize: 'medium' })
-      expect(wrapper.find(AboutSidebar)).to.have.lengthOf(1)
+    it('should pass a navLinks array for pages with content (always Research and Team)', function () {
+      const { getByTestId } = render(
+        <Provider store={mockStore}>
+          <Grommet theme={zooTheme} themeMode='light'>
+            <ProjectAboutPageConnector pageType='team' />
+          </Grommet>
+        </Provider>
+      )
+      // AboutDropdown exists because default screen size is small
+      const dropdown = getByTestId('about-pages-dropdown')
+      expect(dropdown).to.exist()
+
+      // This a side effect behavior of clicking a Grommet DropButton
+      window.scrollTo = () => {}
```

Comment:
Same comment here about setting this up in a `before` and cleaning up in an `after`